<ui version="4.0" >
 <class>aboutBox</class>
 <widget class="QWidget" name="aboutBox" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>About Rockbox Utility</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="rbutilqt.qrc" >:/icons/icons/rockbox-1.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="rbutilqt.qrc" >:/icons/icons/rockbox-5.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTitle" >
       <property name="text" >
        <string>The Rockbox Utility</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QLabel" name="labelText" >
     <property name="text" >
      <string>Installer and housekeeping utility for the Rockbox open source digital audio player firmware.

Â© 2005 - 2007 The Rockbox Team.
Released under the GNU General Public License v2.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="openExternalLinks" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QLabel" name="labelHomepage" >
     <property name="text" >
      <string>http://www.rockbox.org</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="openExternalLinks" >
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags" >
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="creditsTab" >
      <attribute name="title" >
       <string>&amp;Credits</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QTextBrowser" name="browserCredits" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="acceptDrops" >
          <bool>false</bool>
         </property>
         <property name="autoFillBackground" >
          <bool>true</bool>
         </property>
         <property name="autoFormatting" >
          <set>QTextEdit::AutoNone</set>
         </property>
         <property name="textInteractionFlags" >
          <set>Qt::NoTextInteraction</set>
         </property>
         <property name="openExternalLinks" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="licenceTab" >
      <attribute name="title" >
       <string>&amp;License</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QTextBrowser" name="browserLicense" >
         <property name="horizontalScrollBarPolicy" >
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode" >
          <enum>QTextEdit::WidgetWidth</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>191</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" >
    <widget class="QPushButton" name="okButton" >
     <property name="text" >
      <string>&amp;Ok</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rbutilqt.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>aboutBox</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>210</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel" >
     <x>210</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
